### PR TITLE
Modal and grid

### PR DESCRIPTION
**Pull request by amirnathani9**
## changelog

- added modal
- added grid

## screenshots
modal
grid

## preview link
[modal preview](https://deploy-preview-22--nomark-ui.netlify.app/components/modal/modal.html)
[grid preview](https://deploy-preview-22--nomark-ui.netlify.app/components/grid/grid.html)

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                <li><a href="../alert/alert.html" class="sidebar-items">Alert</a></li>
+                <li><a href="../badge/badge.html" class="sidebar-items">Badge</a></li>
+                <li><a href="../button/button.html" class="sidebar-items">Button</a></li>
+                <li><a href="../card/card.html" class="sidebar-items">Card</a></li>
+                <li><a href="../image/image.html" class="sidebar-items">Image</a></li>
+                <li><a href="../input/input.html" class="sidebar-items">Input</a></li>
+                <li><a href="../typography/typography.html" class="sidebar-items">Typography</a></li>
+                <li><a href="../list/list.html" class="sidebar-items">Lists</a></li>
+                <li><a href="../navbar/navbar.html" class="sidebar-items">Navigation</a></li>
+                <li><a href="./model.html" class="sidebar-items active">Modal</a></li>
```
Comment (sharmayatendra) [inline components/modal/modal.html:46]: plz, check the name once here href is "model.html"

**Review by sharmayatendra — APPROVED** [review]
try once checking the comment. Great work

File: components/modal/modal.html (hunk 1 above)
Comment (amirnathani9) [reply components/modal/modal.html:46]: fix the file path

**Pull request merged**